### PR TITLE
allow sending directly to a channel or content address

### PR DESCRIPTION
**Pull request by saltrafael**
## PR Checklist

Please check all that apply to this PR using "x":

- [X] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [ ] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

## Fixes

Issue Number: #5985

## What is the current behavior?

Just wallet address:

## What is the new behavior?

Wallet address + channel or content handle:

### CODE REVIEW

**Comment by jessopb** [comment]
Great feature! 
Check out the tip modal and maybe use two tabs: [ address ] [ lbry:// ] for the different modes,
Then it would be good to show a preview of the selected if you're in url mode as shown here:
https://odysee.com/$/repost?to=jsojsjjjjjj
Check out how this was done in https://github.com/lbryio/lbry-desktop/blob/master/ui/page/repost/view.jsx 

If you're feeling awesome, make pull down autocomplete suggestions :)

**Comment by saltrafael** [comment]
@jessopb thanks for the feedback! something like this?

**Comment by jessopb** [comment]
Great job - I think it looks a bit better with ClaimPreview type="small" in this case - it hides the tags and such. Will take a look at actual code today.

**Comment by jessopb** [comment]
Nice.
 - it doesn't currently work yet - send stays grayed out.
 - It doesn't clear when switching between modes.
  - Might have to change the preview onClick so it doesn't navigate.
  - We are trying to move tab buttons to the card body rather than the title area (I know the tip modal isn't that way yet).
  - i found the input a bit buggy, either refusing to let me edit, etc
  - no error info is being shown during entry
  - feels weird to fill in part of the field, and click send, so the field is more of a search field than the actual entry. 
     -- maybe put a search field on top, the preview in the middle, and the amount and actual recipient url at the bottom?

**Comment by saltrafael** [comment]
check it out now

**Comment by jessopb** [comment]
I think I would use claim preview type small in the confirm too.
And the placeholder text is wrong.
Anything else you think has bad visual hierarchy or spacing, but looks pretty good.

**Comment by saltrafael** [comment]
@jessopb What about this?



> Anything else you think has bad visual hierarchy or spacing, but looks pretty good.

I didn't understand this part

File: ui/page/send/view.jsx [hunk 1]
```diff
@@ -103,7 +103,7 @@ export default function SendPage(props: Props) {
         setContentUri(``);
       }
     }
-  }, [enteredContent, setContentUri, setContentError, parseURI, isNameValid]);
```
Comment (tzarebczan) [inline ui/page/send/view.jsx]: are you sure these two can be removed?

Comment (saltrafael) [reply ui/page/send/view.jsx]: thanks, it was an accident

**Pull request merged**